### PR TITLE
UHM-6175, upload multiple pdf files

### PR DESCRIPTION
**Pull request by cioan**

### CODE REVIEW

File: omod/src/main/webapp/pages/labtrackingAddOrder.gsp [hunk 1]
```diff
@@ -56,7 +56,8 @@ ${ ui.includeFragment("coreapps", "patientHeader", [ patient: patient ]) }
             <div class="form-group row">
                <label class="col-sm-2 col-form-label" for="location">${ui.message("labtrackingapp.orderdetails.locationlabel")}</label>
                <div class="col-sm-9">
-                  <select class="form-control" id="location" type="text" ng-model="order.locationWhereSpecimenCollected"  ng-options="a as a.label for a in locations | orderBy:'label' track by a.value" class="form-control"></select>
+                  <select ng-if="!order.uuid" class="form-control" id="location" type="text" ng-model="order.locationWhereSpecimenCollected"  ng-options="a as a.label for a in locations | orderBy:'label' track by a.value" class="form-control"></select>
+                  <p ng-if="order.uuid" class="form-control-static" > {{ order.locationWhereSpecimenCollected.label }}</p>
```
Comment (mogoodrich) [inline omod/src/main/webapp/pages/labtrackingAddOrder.gsp:60]: This seems unrelated, so I assume an unrelated fix?  (just wanted to make sure it wasn't added by mistake)

**Review by mogoodrich — APPROVED** [review]
LGTM, one small question @cioan , thanks!

File: omod/src/main/webapp/pages/labtrackingAddOrder.gsp (hunk 1 above)
Comment (cioan) [reply omod/src/main/webapp/pages/labtrackingAddOrder.gsp]: Thanks @mogoodrich ! I have removed this line.

**Comment by cioan** [comment]
Thanks! I am going to merge this in, so that we would have time today to test these changes on the humci.
